pkg/upgrade/upgrades/v24_1_drop_payload_and_progress_jobs.go

```diff
@@ -14,26 +14,34 @@ import (
 	"context"
 
 	"github.com/cockroachdb/cockroach/pkg/clusterversion"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog/descs"
 	"github.com/cockroachdb/cockroach/pkg/sql/sessiondata"
 	"github.com/cockroachdb/cockroach/pkg/upgrade"
 )
 
-func dropPayloadProgressFromSystemJobs(
+func hidePayloadProgressFromSystemJobs(
```

Comment:
super nit: we could rename the file name as well. i have no strong opinion this, but just pointing it out since you renamed the function.